src/Microsoft.Diagnostics.Runtime/src/DataTargets/ClrInfo.cs

```diff
@@ -26,122 +31,17 @@ public class ClrInfo : IComparable
         /// </summary>
         public ClrFlavor Flavor { get; }
 
-        /// <summary>
-        /// Returns module information about the Dac needed create a ClrRuntime instance for this runtime.
-        /// </summary>
-        public DacInfo DacInfo { get; }
+        public Architecture Architecture { get; }
+
+        public bool IsLinux { get; }
 
         /// <summary>
         /// Returns module information about the ClrInstance.
         /// </summary>
         public ModuleInfo ModuleInfo { get; }
 
-        /// <summary>
-        /// Returns the location of the local dac on your machine which matches this version of Clr, or null
-        /// if one could not be found.
-        /// </summary>
-        public string LocalMatchingDac { get; }
```

Comment:
I don't believe the DacFileName/DacRequestFileName options provide this info as well, this would be removing functionality, but something the user could theoretically implement themselves.